test/linking_tests.bzl

```diff
@@ -123,6 +132,22 @@ def linking_test_suite(name):
         target_under_test = "//test/test_data:macos_binary",
     )
 
+    opt_osize_enabled_test(
```

Comment:
Can you also add a test that disables the feature?

Reply:
Done